pkg/cmd/init/exec.go

```diff
@@ -23,8 +23,10 @@ func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
 	klog.V(1).InfoS("init options:", "dry-run", o.ClusteradmFlags.DryRun, "force", o.force, "output-file", o.outputFile)
 	o.values = Values{
 		Hub: Hub{
-			TokenID:     helpers.RandStringRunes_az09(6),
-			TokenSecret: helpers.RandStringRunes_az09(16),
+			TokenID:           helpers.RandStringRunes_az09(6),
+			TokenSecret:       helpers.RandStringRunes_az09(16),
+			ReleaseVersion:    o.version,
```

Comment:
move ReleaseVersion ImageRegistryName to another struct

Reply:
factored out a `Image` struct